share/picongpu/unit/metadata/metadataCLI.cpp

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2023 Julian Lenz
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <pmacc/boost_workaround.hpp>
+
+#include "picongpu/ArgsParser.hpp"
+#include "picongpu/MetadataAggregator.hpp"
+
+#include <boost/program_options/options_description.hpp>
+
+#include <algorithm>
+#include <filesystem>
+#include <string>
+
+#include <catch2/catch_test_macros.hpp>
+
+using boost::program_options::options_description;
```

Comment:
in the test cases this is fine because this will not propagate into picongpu, it stays into the cpp file